beets/dbcore/queryparse.py

```diff
@@ -24,6 +24,10 @@
 PARSE_QUERY_PART_REGEX = re.compile(
     # Non-capturing optional segment for the keyword.
     r'(?:'
+    ur'(-|\u00ac)'  # Negation prefixes
```

Comment:
Thanks, I initially tried to remove the `u` indeed but it was causing failures with the prefix tests, causing the `¬` not to be detected. Upon closer inspection, it looks like I was bitten by a unicode gotcha (comparing with `\xac` without `u` works), but in the end I'm keen on the caret as a symbol so will just replace it and forget about `¬` entirely.